packages/ckeditor5-package-tools/lib/tasks/update-pkg-json-main.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license Copyright (c) 2020-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const fs = require( 'fs' );
+
+module.exports = async options => {
```

Comment:
Any reason for marking the function as `async` if inside I do not see any async call?

Reply:
CKE5-pkg-gen depends on `ckeditor5-dev-utils` which exposes util for updating JSON files. Could we re-use it?

Please, take a look at - https://github.com/ckeditor/ckeditor5-dev/blob/6f45c8691deb151b1e7fa37a07070a019c664ae9/packages/ckeditor5-dev-utils/lib/tools.js#L138-L152.